### ISSUE TITLE
Introduce LogService

### ISSUE THREAD

**Issue by DenisVuyka** [Alfresco/alfresco-ng2-components#1152]
**Type of issue:**  (check with "[x]")
```
- [x] New feature request
- [ ] Bug  
- [ ] Support request
```
**Current behavior:**

For the time being various components use `console.log` for debug information output.

**Expected behavior:**

**Steps to reproduce the issue:**

**Component name and version:** 

Should be part of the `ng2-alfresco-core` component.

**Browser and version:** 

 
**Node version (for build issues):**  

**New feature request:**

Components should be injecting `LogService` and using dedicated service instead. It should be possible controlling the level of output (or switching it off) when running with production mode enabled. Service should also improve unit testing process.

For more details refer to the following issue: https://github.com/angular/angular/issues/5458

**Comment by skdhir** [comment]
issue is closed means it is available in current Angular version? sorry not very clear to me

**Comment by DenisVuyka** [comment]
@skdhir every issue has a milestone assigned (right side of the page), this particular feature will be available with `1.1.0` version.

**Comment by dtesta** [comment]
@skdhir I noticed you posted here and in angular/angular#5458 - are you asking about a LogService in Angular or one that is part of the Alfresco Component Library?
- Alfresco Component Library: Implemented
- Angular: No implemented
